docs/README.md

```diff
@@ -1,43 +1,78 @@
 
-### Create pages and posts with the block editor
+### Navigating this handbook
+This handbook is neatly divided into five separate sections each of which serves a different purpose:
 
-In the Block Editor Handbook, our tutorials will be development-focussed. However, it helps if you have some experience using the block editor the way an end-user would first. If you have no experience building with the block editor yet, we recommend you [learn to use the block editor](https://wordpress.org/documentation/article/wordpress-block-editor/) to create posts and pages.
+- **[Getting Started](https://developer.wordpress.org/block-editor/getting-started/)** - for those just starting out with block development this is where you can get set up with a [development environment](https://developer.wordpress.org/block-editor/getting-started/devenv/) and learn the [fundamentals of block development](https://developer.wordpress.org/block-editor/getting-started/create-block/).
+- **[How-to Guides](https://developer.wordpress.org/block-editor/how-to-guides/)** - here you can build on what you learned in the _Getting Started_ section and learn how to solve particular problems that you might encounter. You can also get tutorials, and example code that you can reuse, for projects such as [building a full-featured block](https://developer.wordpress.org/block-editor/how-to-guides/block-tutorial/) or [working with WordPress’ data](https://developer.wordpress.org/block-editor/how-to-guides/data-basics/). In addition you can learn [How to use JavaScript with the Block Editor](https://developer.wordpress.org/block-editor/how-to-guides/javascript/).
+- **[Reference Guides](https://developer.wordpress.org/block-editor/reference-guides/)** - this section is the heart of the handbook and is where you can get down to the nitty-gritty and look up the details of the particular API that you’re working with or need information on. Among other things, the [Block API Reference](https://developer.wordpress.org/block-editor/reference-guides/block-api/) covers most of what you will want to do with a block, and each [component](https://developer.wordpress.org/block-editor/reference-guides/components/) and [package](https://developer.wordpress.org/block-editor/reference-guides/packages/) is also documented here. For components there is also the [Storybook tool](https://wordpress.github.io/gutenberg/?path=/story/docs-introduction--page).
+- **[Explanations](https://developer.wordpress.org/block-editor/explanations/)** - this section enables you to go deeper and reinforce your practical knowledge with a theoretical understanding of the [Architecture](https://developer.wordpress.org/block-editor/explanations/architecture/) of the block editor. There’s also a useful [Glossary of terms](https://developer.wordpress.org/block-editor/explanations/glossary/), and the [FAQs](https://developer.wordpress.org/block-editor/explanations/faq/) should answer any outstanding questions you may have.
```

Comment:
At first glance, it is hard to understand what Explanations are, considering there are already fundamentals linked in the first section.

As far as I understand, this document follows the [Divio Documentation System](https://documentation.divio.com/). Therefore, wouldn't the content of getting started belong to `How-tos` (how to set a dev environment) and `Explanations` (fundamentals of blocks)?